be/src/vec/core/block.h

```diff
@@ -129,6 +129,7 @@ class Block {
 
     Names get_names() const;
     DataTypes get_data_types() const;
+    DataTypePtr get_data_type(size_t index) const { return index < data.size() ? data[index].type : nullptr; }
```

Comment:
Better to use `CHECK(index < data.size())`?

Reply:
OK, i will do what you said, removing condition check will execute quicker